### ISSUE TITLE
Image is not uploading[v4 beta.11]

### ISSUE THREAD

**Issue by amjadniazi48** [strapi/strapi#11486]
### Describe the bug

Image is not uploading in beta.11. I tried to load single image but it failed with status code 400 as depicted in the screenshot below

### Steps to reproduce the behavior

Make any collection type and upload  a single image with media input filed

### Expected behavior

Image must load properly

### Screenshots

![image_uploading](https://user-images.githubusercontent.com/2420029/140561424-ec7f9b88-30ac-4029-9d60-8818288dfb9d.png)

### Code snippets

If applicable, add code samples to help explain your problem.

### System

- Node.js version: 14
- NPM version: 7.5.2
- Strapi version: v4 beta.11
- Database: Postgres
- Operating system: windows 10

**Comment by gat786** [comment]
Did you see some errors in the `console log` when the upload failed? That might give some extra ideas why it did happen.

**Comment by amjadniazi48** [comment]
It shows following errors in browser's console while npm console doesnot show any error

![console](https://user-images.githubusercontent.com/2420029/140714609-a6076a35-2f03-4ff3-a4e4-418c500a4776.png)

**Comment by simonpeters** [comment]
Having the same issue

**Comment by derrickmehaffy** [comment]
Confirmed on beta.11:

![image](https://user-images.githubusercontent.com/8593673/140854179-e1274c41-3086-4189-abe2-c2ce6fdf0d76.png)
![image](https://user-images.githubusercontent.com/8593673/140854200-cc2e1225-c892-4eb5-b5a9-6ef3c89b25f4.png)

```json
{
  "data": null,
  "error": {
    "status": 400,
    "name": "ValidationError",
    "message": "fileInfo must be a `object` type, but the final value was: `\"{}\"`.",
    "details": {
      "errors": [
        {
          "path": ["fileInfo"],
          "message": "fileInfo must be a `object` type, but the final value was: `\"{}\"`.",
          "name": "ValidationError"
        }
      ]
    }
  }
}
```

Example file used:
![images-9 fit_lim size_2000x v1611705151](https://user-images.githubusercontent.com/8593673/140854303-65c69f32-f431-4621-ab97-d6a860b0f69f.jpg)

**Comment by soupette** [comment]
I am closing this issue has it been fixed with #11495
